Update Configuring_a_page_to_update_parameters_on_session_variable_ch… (#4690)
SkylineCommunications/dataminer-docs: user-guide/Basic_Functionality/Visio/session_variables/Configuring_a_page_to_update_parameters_on_session_variable_changes.md

Set|218/652|1203|[var:Chain]|SetTrigger=ValueChanged-
Set|218/652|1205|[var:LoadTime]|SetTrigger=ValueChanged
```

> [!TIP]

> > Change by default separators to avoid having problems with dynamic values (such as Element and View names or properties).

Reviewer: OK to change this like this?
```suggestion
> To avoid problems with dynamic values (such as element and view names or properties), you can change the default separators.
```

Author: I believe many mistakes have been made with separators and therefore I wanted to **politely push people** (i.e. can -> should) to change the separators by default to use a separators that are either exceptional/exotic  (e.g. '§', '@' or 'ùµ' (may be multiple characters))  or forbidden in the dynamic values ([Forbidden characters | DataMiner Docs](https://docs.dataminer.services/user-guide/Reference/Naming_Conventions/Forbidden_characters.html?q=forbidden)) (e.g. '?').

Reviewer: In that case, how about this alternative?
```suggestion
> To avoid problems with dynamic values (such as element and view names or properties), we highly recommend changing the default separators.
```

Reviewer: OK, how about this? 
```suggestion
> To avoid problems with dynamic values (such as element and view names or properties), we highly recommend changing the default separators. Ideally, you should use separators that are exceptional or combine multiple characters, so that they definitely will not occur in the dynamic values, such as `§`, `@`, or `ùµ`, or [forbidden characters](xref:Forbidden_characters) such as `?`.
```
